src/packagedcode/conda.py

```diff
@@ -107,33 +125,152 @@ def parse(cls, location, package_only=False):
             # u'progressbar2', u'python >=3.6'])])
             for req in reqs:
                 name, _, requirement = req.partition(" ")
-                purl = PackageURL(type=cls.default_package_type, name=name)
+                version = None
+                if requirement.startswith("=="):
```

Comment:
I guess this is OK as a quick fix, but we should steal the code from https://github.com/conda/conda/blob/5eed0193f2acd45b26599af4b71f241fd8666ccd/conda/models/version.py#L52 in the future